index.html

```diff
@@ -3443,155 +3470,286 @@ <h2>Processing a Manifest</h2>
+								<p><a href="https://infra.spec.whatwg.org/#map-iterate">For each</a>
+									<var>term</var> → <var>termValue</var> in <var>processed</var>:</p>
+								<ol>
+									<li id="processing-checks-values">
+										<p>(<a href="#properties-value-categories"></a>) If <var>term</var> has a known
+												<a href="#properties-value-categories">value type</a>, and
+												<var>termValue</var> does not match the expected type, this is a
+												<a>validation error</a></p>
+									</li>
+
+									<li id="processing-checks-entities">
+										<p>(<a href="#value-entity"></a>) If <var>term</var> expects an <a
+												href="#value-array">array</a> of <a href="#value-entity">entities</a>,
+											check whether each <a href="https://infra.spec.whatwg.org/#ordered-map"
+												>map</a> in <var>termValue</var> has a <var>name</var>
+											<a href="https://infra.spec.whatwg.org/#map-key">key</a>. Each item without
+											a <var>name</var> is a <a>validation error</a>.</p>
+									</li>
+
+									<li id="processing-checks-language">
+										<p>(<a href="#manifest-lang-dir-global"></a> and <a
+												href="#manifest-lang-dir-local"></a>) If <var>termValue</var> is a <a
+												href="https://infra.spec.whatwg.org/#ordered-map">map</a>:</p>
+										<ul>
+											<li>
+												<p>if it has a <var>language</var>
+													<a href="https://infra.spec.whatwg.org/#map-key">key</a>, and
+														<var>termValue["language"]</var> is not <a
+														href="https://tools.ietf.org/html/bcp47#section-2.2.9"
+														>well-formed</a>&#160;[[!bcp47]], this is a <a>validation
+														error</a>.</p>
+												<p>if it has a <var>direction</var>
+													<a href="https://infra.spec.whatwg.org/#map-key">key</a>, and
+														<var>termValue["direction"]</var> is not one of
+														"<code>ltr</code>" or "<code>rtl</code>", this is a
+														<a>validation error</a>.</p>
+											</li>
+										</ul>
+									</li>
+
+									<li id="processing-checks-urls">
+										<p>(<a href="#resource-categorization-properties"></a>) If <var>processed</var>
+											is a <a href="#resource-categorization-properties">resource categorization
+												property</a>, check that each item in <var>termValue</var>:</p>
+										<ul>
+											<li>
+												<p>if it does not have a <var>url</var>
+													<a href="https://infra.spec.whatwg.org/#ordered-map">key</a>, this
+													is a <a>validation error</a>.</p>
+											</li>
+											<li>
+												<p>otherwise, if <var>url</var> is not a valid URL&#160;[[!url]], this
+													is a <a>validation error</a>.</p>
+											</li>
+										</ul>
+									</li>
+
+									<li id="processing-checks-length">
+										<p>(<a href="#LinkedResource"></a>) If <var>processed[term]</var> expects type
+												<a>LinkedResource</a> and its <var>termValue["length"]</var>
```

Comment:
This links to the WebIDL dictionary, which is problematic. We do not know if WebIDL will survive and, by now, the term "dictionary" has become banned:-)

It should probably link too 2.3.4.3., ie, `#value-linked-resource` instead.

Reply:
This is not yet done in the latest incarnation...

Reply:
(I mean in point 8.1.5)

Reply:
Oops, ya, I fixed the second instance where LinkedResource was tagged, but didn't get the reference in parens. Will patch in a minute with some other tweaking of the references to linkedresource in the doc.